app/overrides/decidim/proposals/proposals/edit_draft/replace_partial_edit_form_fields.html.erb.deface

```diff
@@ -0,0 +1,3 @@
+<!-- replace "erb[loud]:contains('edit_form_fields')" -->
+
+<%= render "decidim/reporting_proposals/proposals/#{reporting_proposal? ? 'reporting_proposal_fields' : 'edit_form'}", form: form %>
```

Comment:
```suggestion
<%= render "decidim/#{reporting_proposal? ? reporting_proposals' : 'proposals'}/proposals/edit_form", form: form %>
```
I think in here we want to use the original form when the proposals is a normal one, and our editor when is a reporting proposal

Reply:
Yes, I corrected a few deface files. Now the original form is not overridden